config/config.rb

```diff
@@ -77,6 +77,21 @@
 
   configatron.taxon_lookup_url = 'http://eutils.ncbi.nlm.nih.gov/entrez/eutils/'
 
+  configatron.amqp.isg.host = 'localhost'
+  configatron.amqp.isg.tls = false
+  configatron.amqp.isg.vhost = 'tol'
+  configatron.amqp.isg.username = 'admin'
+  configatron.amqp.isg.password = 'development'
+  configatron.amqp.isg.exchange = 'sequencescape'
+
+  configatron.amqp.schemas.registry_url = 'http://redpanda.uat.psd.sanger.ac.uk/subjects/'
```

Comment:
Do we want local config to be dependent on the UAT instance? Maybe it's not dependent...

Reply:
I had considered this when I made this choice and I don't know that there are any simple and easy answers.  If I make it dependent on a local instance of RedPanda, that adds dependencies to Sequencescape when we already have so many.  Although thinking about it again, you would only be able to run a local instance while on the VPN when you rely on UAT's RedPanda.  Thoughts?

Reply:
If you don't have a connection to UAT's RedPanda, what will happen? If it breaks Sequencescape completely, I think that's not OK. If it just means this particular feature doesn't work, that's not too bad. 

Similarly, I don't want a local RedPanda instance to be a requirement for developing SS locally... but if it was required for just this feature, wouldn't be too bad.

Would either of these work?
1. Skip the check against the schema if one is not available? (in local only)
2. Have a dummy schema in a file in the codebase that it uses in local only

Reply:
I've added a cache to the system, which it probably should have had to start with.  This means that you would need to connect to the UAT RedPanda the first time you trigger one of these jobs, but then you'd have a cached file of the schema to use in future calls.  Of course, if you know the schema, you can also just dump it in the appropriately named cache file.  I added some info to the README about this.